docs/src/main/asciidoc/_elytron/Credential_Store.adoc

```diff
@@ -3,17 +3,19 @@
 
 One new component included with WildFly Elytron for the secure storage of credentials is the Credential Store.  Previous versions of the application server made use of the Vault which was used for the secure storage of clear text strings; the credential store moves forward a step to focus on the secure storage of credentials.  As with the Vault the stored credentials could be clear text passwords however other formats are also supported.
 
-WildFly Elytron contains a default credential store implementation `KeyStoreCredentialStore` which is a store implementation backed by a `KeyStore`.
+WildFly Elytron contains two default credential store implementations.  The first implementation is the `KeyStoreCredentialStore` which is an
+implementation backed by a `KeyStore`.  The `KeyStoreCredentialStore` implementation supports the storage of various credential types,
+`PasswordCredential`, `KeyPairCredential`, and `SecretKeyCredential`.  The second implementation is the `PropertiesCredentialStore` this credential
+store is dedicated to the storage of `SecretKeyCredential` instances using a properties file.  The `PropertiesCredentialStore` does not offer any protection
+of the credentials it stores so it's primary purpose is to provide an initial key to a server environment.
 
-This documentation will be focused on using the `KeyStoreCredentialStore`; however the section <<Custom_CredentialStore, Custom CredentialStore>> will describe how to convert a vault to a credential store. 
+This documentation is primarily focussed on the `KeyStoreCredentialStore` and `PropertiesCredentialStore`; however the section <<Custom_CredentialStore, Custom CredentialStore>> describes the SPIs for implementatin a custom credential store and the section <<Migrating_Existing_Vaults, Migrating Existing Vaults>> describes how to convert a vault to a credential store.
```

Comment:
https://proofreadmyessay.co.uk/writing-tips/spelling-tips-focused-or-focussed/

Up to you. :)

Reply:
s/implementatin/implementing